docs/locales/fr/LC_MESSAGES/authors.po

```diff
@@ -3,21 +3,8 @@
 # This file is distributed under the same license as the xHydro package.
 # FIRST AUTHOR <EMAIL@ADDRESS>, 2025.
 #
-#, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: xHydro 0.4.2\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2025-04-03 11:23-0400\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language: fr\n"
-"Language-Team: fr <LL@li.org>\n"
-"Plural-Forms: nplurals=2; plural=(n > 1);\n"
-"MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=utf-8\n"
-"Content-Transfer-Encoding: 8bit\n"
-"Generated-By: Babel 2.17.0\n"
```

Comment:
Any reason why you're removing this? Many of the metadata fields here are useful to have (`Content`, `Generated-By`, `Revision-Date`).

Reply:
If you look at https://github.com/hydrologie/xhydro/pull/293/commits/834050c6c2b996e51ab572d0044d9df7e98b8bb3, these entries have a ", fuzzy" tag, so they got deleted by the `translator.py` script. I can try to restore them.